frontend/src/bundles/chat/components/message-input/message-input.tsx

```diff
@@ -0,0 +1,78 @@
+import { IconButton, Input as MuiInput, InputAdornment } from '@mui/material';
```

Comment:
Input can be changed to frontend/src/bundles/common/components/input?

Reply:
No. I checked this possibility and decided that common input is not appropriate solution. Common input component serves like a part of the form so it has a lot of logic that relatives to forms (validation, errors, helpers text, different types) and actually that is not Input component but TextField. As MUI documentation says "TextField is composed of smaller components ( FormControl, Input, FilledInput, InputLabel, OutlinedInput, and FormHelperText ) and from all these parts for chat we need only Input fuctionalities.

Reply:
Could we use common TextArea component for it?